pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -142,6 +142,12 @@ func (c *cephStatusChecker) checkStatus() {
 		if err := c.forceDeleteStuckRookPodsOnNotReadyNodes(); err != nil {
 			logger.Errorf("failed to delete pod on not ready nodes. %v", err)
 		}
+
+		if osd_Nearfull, ok := status.Health.Checks["OSD_NEARFULL"]; ok {
+			if osd_Nearfull.Severity == "HEALTH_WARN" || osd_Nearfull.Severity == "HEALTH_ERR" {
```

Comment:
Will the nearfull severity ever be error? I thought it would be a warning.

Reply:
In addition to the nearfull warning, we should probably also check for the health error that OSD(s) have hit the full ratio.

Reply:
Yes, it's an error string and a warning is a `summary` of [CheckMessage](https://github.com/rook/rook/blob/b37e620549871d612261b1bb0cb65febe51ad5a2/pkg/daemon/ceph/client/status.go#L64). Okay Added the check for OSD_FULL.